app/services/CartService.java

```diff
@@ -0,0 +1,25 @@
+package services;
+
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.products.ProductProjection;
+import io.sphere.sdk.products.ProductVariant;
+import pactas.models.PactasContract;
+import pactas.models.PactasCustomer;
+import play.mvc.Http;
+
+import java.util.Optional;
+
+public interface CartService {
```

Comment:
Interfaces should be well documented. 

examples:
* https://github.com/sphereio/sphere-sunrise/blob/3e2ec4a761325d16558177354162d6288648f093/move-to-sdk/app/io/sphere/sdk/facets/FacetOptionMapper.java
* https://github.com/sphereio/sphere-sunrise/blob/027c73052f687c240e50f1f5f3dd5459a2903cd8/common/app/common/cms/CmsService.java
* https://github.com/sphereio/sphere-sunrise/blob/3e2ec4a761325d16558177354162d6288648f093/move-to-sdk/app/io/sphere/sdk/facets/Facet.java